src/main/java/org/spdx/sbom/gradle/SpdxPackagePlugin.java

```diff
@@ -27,5 +27,6 @@ public void apply(@NotNull Project project) {
     extension.getName().convention(project.getName());
```

Comment:
Hmm I'm not sure how to leave a comment on the commit message so I'm just leaving it here for now.

Do we know how the plan to read spdx info from subprojects will interact with Gradle project isolation https://gradle.github.io/configuration-cache/#project_isolation ?

My guess is that probably there was already something incompatible previously and that this will still be incompatible?

Oh - I think I see that the SpdxPackagePlugin is a new plugin that allows a project to specify some spdx information. I guess this means that this change isn't specifically for reading subproject information but for reading information from any dependency project (which could also be a sibling project) - right?

Reply:
It seems like this would not work with project isolation

Reply:
You still need to operate on the sub projects configuration

Reply:
I think project isolation would break this plugin completely though

Reply:
You can file a gradle issue where you can put your use case that would break project isolation and they'll take it into consideration when developing APIs for accessing things like this in a compliant way. 

I think it's possible to do it with existing APIs though. Haven't fully thought it out but a shared build service might be the answer.

Reply:
It looks like the suggestion is to create another configuration that outputs the spdx info for a project and consume that artifact if necessary. This might require a pretty major rework. I'll have to look into it a bit. I think we kinda move forward with this for now though